tests/app/dao/test_provider_details_dao.py

```diff
@@ -19,7 +19,7 @@
 )
 
 
-def set_primary_sms_provider(identifier='mmg'):
+def set_primary_sms_provider(restore_provider_details, identifier='mmg'):
```

Comment:
dont think this is meant to be here - this isn't a fixture, and so passing this in isn't doing what you think it should do

also i don't think identifier should have a default value - we don't call this function without specifying the provider